eras/conway/impl/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.8.0.0
 
+* Change `Environment (ConwayENACT era)` to `GovActionId (EraCrypto era)`
```

Comment:
`docs/diagrams` in the other PR needs a corresponding update :slightly_smiling_face: